### DISCUSSION TITLE
Keyerror 600

### DISCUSSION THREAD

**Discussion by shividwivedi** [Kludex/uvicorn#2465] [Q&A] [answered]
i want to send 600 as my status code but getting error! is there any custom way to do this!. it is required for me to send this 600

**Comment by Kludex** [comment]
I think with `--http httptools` you can do that... Or maybe it's with `--http h11`... I don't know.

**Reply by Kludex** [reply, marked as answer]
Yeah, it looks like we limit the range of status code from 100 to 599. Which makes sense according to the HTTP specification.

**Reply by shividwivedi** [reply]
Okay so ! It is not possible? . I should limit it between 100 to 599 ?

**Reply by Kludex** [reply]
Yes. Every status code has a meaning: https://www.webfx.com/web-development/glossary/http-status-codes/
